### PR TITLE
UHM-3660 : get patient-uuid from url params

### PR DESCRIPTION
**Pull request by rhenshaw56**

### CODE REVIEW

File: app/js/components/LabResultsList.jsx [hunk 1]
```diff
@@ -129,16 +130,23 @@ export class LabResultsList extends PureComponent {
...
-    dispatch(constantsActions.fetchLabResultsDateConcept());
-    dispatch(constantsActions.fetchLabResultsDidNotPerformQuestion());
-    dispatch(constantsActions.fetchLabResultsDidNotPerformReasonQuestion());
-    dispatch(constantsActions.fetchLabResultsTestOrderNumberConcept());
-    dispatch(constantsActions.fetchLabResultsTestLocationQuestion());
-    dispatch(constantsActions.fetchLabResultsEstimatedCollectionDateQuestion());
-    dispatch(constantsActions.getDateAndTimeFormat());
-    dispatch(patientAction.getPatient(patientUUID));
-    dispatch(patientAction.fetchPatientLabTestResults(patientUUID));
+    const params = qs.parse(decodeURIComponent(window.location.search));
```
[the remark this answers is not in the document]
Comment (jomadoye) [reply app/js/components/LabResultsList.jsx]: const patient =  new URLSearchParams(this.props.location.search).get('patient'),

Comment (jomadoye) [inline app/js/components/LabResultsList.jsx]: const patient = new URLSearchParams(this.props.location.search).get('patient'),

[the remark this answers is not in the document]
Comment (rhenshaw56) [reply app/js/components/LabResultsList.jsx]: Thanks @jomadoye

[the remark this answers is not in the document]
Comment (rhenshaw56) [reply app/js/components/LabResultsList.jsx]: @jomadoye In this case, I don't think we need to trust `this.props.location.search`

**Pull request merged**